specifyweb/frontend/js_src/lib/components/FormEditor/Editor.tsx

```diff
@@ -197,10 +196,11 @@ function UseLabelsSchema(): JSX.Element {
   };
 
   return (
-    <Label.Inline>
-      <Input.Checkbox checked={useFieldLabels} onValueChange={update} />
-      {formsText.useFieldLabels()}
-    </Label.Inline>
+    <Button.Secondary onClick={update}>
+      {useFieldLabels
```

Comment:
Shouldn't these labels be swapped?

i.e if we currently use localized field labels, then the button should say `Use Database Field Names` so that the user knows what action to expect when they click the button

Reply:
yes indeed!